### PR TITLE
Remove flaky check for analyzer message.

### PR DESCRIPTION
**Pull request by sjindel-google**
## Description

Remove the check on a word which the analyzer has changed.

### CODE REVIEW

**Review by mkustermann — APPROVED** [review]
For others: The analyzer warning message change will cause this test to fail when the roll happens due to some minor message renaming. Though our head-head-head builder fails already now due to it. This will make the test more robust to handle both cases (the old error message and the new one).

**Comment by sjindel-google** [comment]
I'm assuming the codecov report is a flake, since this change could not possibly have affected control flow.

File: packages/flutter_tools/test/general.shard/commands/analyze_once_test.dart [hunk 1]
```diff
@@ -99,7 +99,7 @@ void main() {
         statusTextContains: <String>[
           'Analyzing',
           'warning $analyzerSeparator The parameter \'onPressed\' is required',
-          'info $analyzerSeparator The method \'_incrementCounter\' isn\'t used',
```
Comment (aam) [inline packages/flutter_tools/test/general.shard/commands/analyze_once_test.dart]: Unfortunately that was not the only change. The word `method` was replaced with word `declaration` as well.
```
   Expected: contains 'info • The method \'_incrementCounter\' isn\'t used'
  Actual: 'Analyzing flutter_project...\n'
            '\n'
            '   info • The declaration \'_incrementCounter\' isn\'t referenced • ../../../../../tmp/t/flutter_analyze_once_test_1.EEEWQC/flutter_project/lib/main.dart:49:8 • unused_element\n'
```

Comment (mkustermann) [reply packages/flutter_tools/test/general.shard/commands/analyze_once_test.dart]: [Siva's roll](https://github.com/flutter/flutter/commit/c15cfdd6af8431bd533cfbf4c195176061b0f680) included that missing change.

**Pull request merged**